docs/WindowsBuild.md

```diff
@@ -1,23 +1,34 @@
 # Building Swift on Windows
 
-Visual Studio 2017 or newer is needed to build swift on Windows.
+Visual Studio 2017 or newer is needed to build Swift on Windows.
 
-The following must take place in the **developer command prompt** (provided by Visual Studio). This shows up as "x64 Native Tools Command Prompt for VS2017" (or VS2019, VS2019 Preview depending on the Visual Studio that you are using) in the Start Menu.
+The commands below (with the exception of installing Visual Studio) must be entered in the "**x64 Native** Tools Command Prompt for VS2017" (or VS2019, VS2019 Preview depending on the Visual Studio that you are using) in the Start Menu. This sets environment variables to select the correct target platform.
 
 ## Install dependencies
 
-- Install the latest version of [Visual Studio](https://www.visualstudio.com/downloads/)
-- Make sure to include "Programming Languages|Visual C++" and "Windows and Web Development|Universal Windows App Development|Windows SDK" in your installation. The following components are required:
+### Visual Studio
 
-1. Microsoft.VisualStudio.Component.Windows10SDK
-2. Microsoft.VisualStudio.Component.Windows10SDK.17763
-3. Microsoft.VisualStudio.Component.VC.Tools.x86.x64
+An easy way to get most of the tools to build Swift is using the [Visual Studio installer](https://www.visualstudio.com/downloads/). This command installs all needed Visual Studio components as well as Python and Git:
+
+```
+vs_community ^
+  --add Component.CPython2.x86 ^
+  --add Component.CPython3.x64 ^
+  --add Microsoft.VisualStudio.Component.Git ^
+  --add Microsoft.VisualStudio.Component.VC.ATL ^
+  --add Microsoft.VisualStudio.Component.VC.CMake.Project ^
+  --add Microsoft.VisualStudio.Component.VC.Tools.x86.x64 ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK ^
+  --add Microsoft.VisualStudio.Component.Windows10SDK.17763
+```
```

Comment:
Hmm, I’d tried that but it seemed to exit and run in the background.  Did that not happen for you?

Reply:
It runs in the background if I specify `-q`, which I didn't. For me it opens the normal installation dialog with all the components preselected.

Reply:
Well, actually, depends on what you mean with "in the background". The command certainly exits on the command line. But it opens a UI.

Reply:
Really?  Okay, nvm then.  I thought that this was batched not interactive.